src/main/java/Chatbot.java

```diff
@@ -0,0 +1,398 @@
+                        Ui.todoMessage(todo, taskList);
+                    } catch (Exception e) {
+                        Ui.formatErrorMessage("todo");
+                    }
+                } else if (userMessage.substring(0, 8).equalsIgnoreCase("deadline")) {
+                    try {
+                        int index = userMessage.indexOf("/by");
+                        String description = userMessage.substring(9, index - 1);
+                        LocalDateTime taskDeadline = LocalDateTime.parse(userMessage.substring(index + 4), formatter);
+                        Deadline deadline = new Deadline(description, taskDeadline);
+                        Ui.deadlineMessage(deadline, taskList);
+                    } catch (Exception e) {
+                        Ui.formatErrorMessage("deadline");
+                    }
+                } else if (userMessage.substring(0, 5).equalsIgnoreCase("event")) {
+                    try {
+                        int index = userMessage.indexOf("/from");
+                        int index2 = userMessage.indexOf("/to");
+                        String description = userMessage.substring(6, index - 1);
+                        String start = userMessage.substring(index + 6, index2 - 1);
+                        String end = userMessage.substring(index2 + 4);
+                        LocalDateTime startTime = LocalDateTime.parse(start, formatter);
+                        LocalDateTime endTime = LocalDateTime.parse(end, formatter);
+                        Event event = new Event(description, startTime, endTime);
+                        Ui.eventMessage(event, taskList);
+                    } catch (Exception e) {
+                        Ui.formatErrorMessage("event");
+                    }
+                } else {
+                    Ui.inputErrorMessage();
+                }
+            } catch (Exception e) {
+                Ui.inputErrorMessage();
+            }
+        }
+    }
+
+    public static class TaskList {
+        private Task[] tasks;
+        private int taskCount;
+
+        public TaskList() {
+            tasks = new Task[100];
+            taskCount = 0;
+        }
+
+        public void addTask(Task task) {
+            tasks[taskCount] = task;
+            taskCount++;
+        }
+
+        public void deleteTask(int task) {
+            Task[] newTasks = new Task[100];
+            for (int i = 0; i < taskCount - 1; i++) {
+                int j = i;
+                if (i >= task) j = i + 1;
+                newTasks[i] = tasks[j];
+            }
+            tasks = newTasks;
+            taskCount--;
+        }
+
+        public void markTask(int task) {
+            tasks[task].markAsDone();
+        }
+
+        public int getCount() {
+            return taskCount;
+        }
+
+        public Task getTask(int i) {
+            return tasks[i];
+        }
+    }
+
+    public static class Storage {
+        public static void load(TaskList taskList) {
```

Comment:
I'm curious as to why load is named as such. Is there any reason why you decided to name it like that instead of loadTasks?